README.md

```diff
@@ -8,14 +8,17 @@ contexts too.
 Installation
 ------------
 
-Once you have [installed Go][golang-install], install the `mockgen` tool:
+Once you have [installed Go][golang-install], install the `mockgen` tool.
+
+To get the latest released version use:
 
 ```bash
-go get github.com/golang/mock/mockgen
+GO111MODULE=on go get github.com/golang/mock/mockgen@latest
```

Comment:
These extra options should not be needed with the current version of go right?

Reply:
Actually it is needed because `GO111MODULE` is by default set to `auto`. That means that if you invoke `go get` in a directory outside the `GOPATH` and without go.mod file, then the `go get` executes in  `GOPATH mode`. These are the docs:
```
 If GO111MODULE=auto or is unset, then the go command enables or disables module support based on the current directory. Module support is enabled only when the current directory contains a go.mod file or is below a directory containing a go.mod file.
```
(source: https://golang.org/cmd/go/#hdr-Module_support)